gon/linear/segment.py

```diff
@@ -3,31 +3,35 @@
 from orient.planar import (point_in_segment,
                            segment_in_segment)
 from reprit.base import generate_repr
+from robust import projection
 from robust.hints import Point
```

Comment:
Why would you import type hints from robust, though?

Reply:
PyCharm can't work with `gon` anymore: eats all of the memory, but still can't correctly remove unused/overwritten imports, that's a shame